scripts/dts/edtlib.py

```diff
@@ -101,8 +101,8 @@ def __init__(self, dts, bindings_dir):
           Path to device tree .dts file
 
         bindings_dir:
```

Comment:
Could call this this `binding_dirs` maybe (`bindings_dirs` is a bit of a mouthfull).

Reply:
Heh... looks like someone else thought `bindings_dirs` was a bit much. `binding_dirs` might be a happy medium.

Reply:
Not if the directory name is `bindings`.

Reply:
I was thinking "binding directories" as in "directories with bindings".

It's pretty meh though. Just put the `s`s where you feel like it. :P